pose-detection/README.md

```diff
@@ -53,28 +53,39 @@ only PoseNet supports multi-pose estimation. If the model cannot detect any pose
 the list will be empty.
 
 For each pose, it contains a confidence score of the pose and an array of keypoints.
-PoseNet and MoveNet both return 17 keypoints. Mediapipe BlazePose returns 33 keypoints.
-Each keypoint contains x, y, score and name.
+PoseNet and MoveNet both return 17 keypoints. MediaPipe BlazePose returns 33 keypoints.
+Each keypoint contains x, y, score and name. In addition, MediaPipe BlazePose
+also returns an array of 3D keypoints.
 
 Example output:
 ```
 [
   {
     score: 0.8,
     keypoints: [
-      {x: 230, y: 220, score: 0.9, name: "nose"},
-      {x: 212, y: 190, score: 0.8, name: "left_eye"},
+      {x: 230, y: 220, score: 0.9, score: 0.99, name: "nose"},
+      {x: 212, y: 190, score: 0.8, score: 0.91, name: "left_eye"},
+      ...
+    ],
+    keypoints3D: [
+      {x: 0.65, y: 0.11, z: 0.05, score: 0.99, name: "nose"},
       ...
     ]
   }
 ]
 ```
 
-x and y represent the actual keypoint position in the image. If you need normalized
-keypoint positions, you can use the method
+For the `keypoints`, x and y represent the actual keypoint position in the image.
+If you need normalized keypoint positions, you can use the method
 `poseDetection.calculator.keypointsToNormalizedKeypoints(keypoints, imageSize)` to
 convert x and y to [0, 1] range.
 
+For the `keypoints3D`, x, y and z represent absolute distance in meters in a
+1 x 1 x 1 meter cubic space. To map the 3D pose to real world size, applications
```

Comment:
As I mentioned in doc, if you are are using the output from MP Solution API, that should be 2x2x2 cube with hips in (0, 0, 0)